WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadViewModelTest.kt

```diff
@@ -34,13 +164,56 @@ class BackupDownloadViewModelTest {
         assertThat(toolbarStates.size).isEqualTo(0)
     }
 
+    @Test
+    fun `backupDownloadState is writtenToBundle`() {
+        viewModel.start(savedInstanceState = null)
+
+        viewModel.writeToBundle(savedInstanceState)
```

Comment:
Minor (🔍): Consider adding an empty line here to split the `Act` from the `Assert`.